syntax = "proto3";

package service;

// Mensaje para la solicitud de lookupID
message LookupIDRequest {
  int32 idNode = 1;  // ID del nodo a buscar
}

// Mensaje para la respuesta de lookupID
message LookupIDResponse {
  bool result = 1;           // Resultado de la búsqueda (booleano)
  string address = 2;        // Dirección del nodo (como string)
}

// Servicio que expone el método lookupID
service Searchsucc {
  rpc LookupID (LookupIDRequest) returns (LookupIDResponse);
}
